docs/core/additional-tools/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Using Microsoft XML Serializer Generator on .NET Core
+description: An overview of the Microsoft XML Serializer Generator.
+author: mlacouture
+manager: wpickett
+ms.author: johalex
+ms.date: 12/20/2017
+ms.topic: tutorial
+ms.prod: .net-core
+ms.custom: mvc
+---
+# Using Microsoft XML Serializer Generator on .NET Core
+
+This tutorial teaches you how to use the Microsoft XML Serializer Generator in a C# .NET Core application. During the course of this tutorial, you learn:
+
+> [!div class="checklist"]
+> * How to create a .NET Core app.
+> * How to add a reference to the Microsoft.XmlSerializer.Generator package
+> * How to edit your MyApp.csproj to add dependencies.
+> * How to add a class and an XmlSerializer
+> * How to build and run the application 
+
+Like the [Xml Serializer Generator (sgen.exe)](https://docs.microsoft.com/en-us/dotnet/standard/serialization/xml-serializer-generator-tool-sgen-exe) for .NET Framework, <xref:System.Xml.Serialization.XmlSerializer> is the equivalent for .NET Core and .NET Standard projects. It creates an XML serialization assembly for types contained in an assembly to improve the startup performance of XML serialization when serializing or de-serializing objects of those types using XmlSerializer.
+
+## Prerequisites
+
+To complete this tutorial:
+
+* Install [.NET Core SDK 2.1.3 or later](https://www.microsoft.com/net/download)
+* Install your favorite code editor, if you haven't already.
+
+> [!TIP]
+> Need to install a code editor? Try [Visual Studio](https://visualstudio.com/downloads)!
```

Comment:
the link used by the vsdocs team is this one: https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs